packages/driver/src/cypress/error_messages.ts

```diff
@@ -1665,6 +1665,32 @@ export default {
     },
   },
 
+  switchToDomain: {
+    experiment_not_enabled: {
```

Comment:
I think this can get hoisted here?
```suggestion
   docsUrl: 'https://on.cypress.io/switchToDomain',
    experiment_not_enabled: {
```